src/c++/examples/simple_http_infer_client.cc

```diff
@@ -82,6 +83,11 @@ Usage(char** argv, const std::string& msg = std::string())
   std::cerr << "\t-i <none|gzip|deflate>" << std::endl;
   std::cerr << "\t-o <none|gzip|deflate>" << std::endl;
   std::cerr << std::endl;
+  std::cerr << "\t--verify-peer" << std::endl;
+  std::cerr << "\t--verify-host" << std::endl;
+  std::cerr << "\t--ca-certs" << std::endl;
+  std::cerr << "\t--cert-file" << std::endl;
+  std::cerr << "\t--key-file" << std::endl;
```

Comment:
how come cert type and key type aren't settable here?

Reply:
It's just a simple example. In order to not pollute it with too many options I decided to not expose the type parameters here. In our test script that use these example we use PEM type which is what is set by default.

Reply:
For someone writing their client with our library we do expose and option for them to specify DER keys.

Reply:
resolved offline